_rules/SC4-1-1-unique-id.md

```diff
@@ -11,20 +11,18 @@ test_aspects:
 
 authors:
 - Bryn Anderson
-- Jey Nandakumar
-- Kamyar Rasta
-- Wilco Fiers
+- Anne Thyme
 ---
 
 ## Test procedure
 
 ### Applicability
 
-Each test target has an `id` attribute that is unique within the [document context](https://github.com/auto-wcag/auto-wcag/pull/110/files#insert-link-here) of the element.
+Any element that has an `id` attribute within a document.
 
 ### Expectation
 
-All `id` attribute values are unique.
+Each test target has an `id` attribute value that is unique within the [document context](#document-context) of the element.
```

Comment:
We should add a note here explaining "document context". Something like:

> **note**: Elements can live in different document contexts if they are part of different shadow DOM trees. Elements not inside shadow DOM are always on the same document context.

Reply:
@WilcoFiers: please note that we have suggested that "document context" should link to a definition, which kind of acts as a note shared across rules...

Reply:
Issue opened for "document context" algorithm: https://github.com/auto-wcag/auto-wcag/issues/154
However, I can see that @jkodu already signed up for giving that algorithm a shot...
@WilcoFiers, can I consider your review comments resolved?